src/main/java/net/neoforged/neoforge/internal/NeoForgeStatesProvider.java

```diff
@@ -20,10 +21,11 @@ public class NeoForgeStatesProvider implements IModStateProvider {
     final ModLoadingState LOAD_REGISTRIES = ModLoadingState.withInline("LOAD_REGISTRIES", "UNFREEZE_DATA", ModLoadingPhase.GATHER, ml -> GameData.postRegisterEvents());
     final ModLoadingState FREEZE = ModLoadingState.withInline("FREEZE_DATA", "LOAD_REGISTRIES", ModLoadingPhase.GATHER, ml -> GameData.freezeData());
     final ModLoadingState REGISTRATION_EVENTS = ModLoadingState.withInline("REGISTRATION_EVENTS", "SIDED_SETUP", ModLoadingPhase.LOAD, ml -> RegistrationEvents.init());
```

Comment:
This needs to then be `CLIENT_REGISTRATION_EVENTS` instead of `SIDED_SETUP`

Reply:
ok

Reply:
fixed!